src/cript/api/api.py

```diff
@@ -643,6 +649,10 @@ def _is_node_schema_valid(self, node_json: str, is_patch: bool = False) -> bool:
             whether the node JSON is valid or not
         """
 
+        # Fast exit without validation
+        if self.skip_validation and not force_validation:
+            return True
```

Comment:
should we just return if we are not validating?
I think this version works, but I wonder if that is more accurate or not, not sure

Reply:
we specify that we return a bool with type hints, so it has to be True.

Reply:
should we change it to returns boolean or none?
I'm thinking that way it doesn't mislead the user if they forget to turn it ON/OFF or something. what do you think?

Reply:
I see what you mean, but I don't think it matters much.
I can implement that.

Reply:
done